src/sentry/static/sentry/app/data/forms/apiApplication.jsx

```diff
@@ -0,0 +1,83 @@
+// import {createSearchMap} from './util';
```

Comment:
why doesn't this go into the search map? should the comment be removed?

Reply:
Seems weird to have the form in search, I think when you're searching you'll be searching for the route and not a specific settings to change?

Reply:
But maybe I'm wrong